src/descriptors/power_system_structs.json

```diff
@@ -1,5034 +1,5077 @@
+          "internal_default": "InfrastructureSystemsInternal()"
+        }
+      ],
+      "supertype": "PSS"
     },
     {
-        "struct_name": "SingleMass",
-        "docstring": "Parameters of single mass shaft model. Typically represents the rotor mass.",
-        "fields": [
-            {
-                "name": "H",
-                "comment": "Rotor inertia constant in MWs/MVA",
-                "null_value": 0,
-                "data_type": "Float64",
-                "valid_range": {
-                    "min": 0,
-                    "max": null
-                }
-            },
-            {
-                "name": "D",
-                "comment": "Rotor natural damping in pu",
-                "null_value": 0,
-                "data_type": "Float64",
-                "valid_range": {
-                    "min": 0,
-                    "max": null
-                }
-            },
-            {
-                "name": "ext",
-                "data_type": "Dict{String, Any}",
-                "null_value": "Dict{String, Any}()",
-                "default": "Dict{String, Any}()"
-            },
-            {
-                "name": "states",
-                "comment": "",
-                "internal_default": "[:δ, :ω]",
-                "data_type": "Vector{Symbol}"
-            },
-            {
-                "name": "n_states",
-                "comment": "",
-                "internal_default": 2,
-                "data_type": "Int64"
-            },
-            {
-                "name": "internal",
-                "comment": "power system internal reference, do not modify",
-                "data_type": "InfrastructureSystemsInternal",
-                "internal_default": "InfrastructureSystemsInternal()"
-            }
-        ],
-        "supertype": "Shaft"
+      "struct_name": "SingleMass",
+      "docstring": "Parameters of single mass shaft model. Typically represents the rotor mass.",
+      "fields": [
+        {
+          "name": "H",
+          "comment": "Rotor inertia constant in MWs/MVA",
+          "null_value": 0,
+          "data_type": "Float64",
+          "valid_range": {
+            "min": 0,
+            "max": null
+          }
+        },
+        {
+          "name": "D",
+          "comment": "Rotor natural damping in pu",
+          "null_value": 0,
+          "data_type": "Float64",
+          "valid_range": {
+            "min": 0,
+            "max": null
+          }
+        },
+        {
+          "name": "ext",
+          "data_type": "Dict{String, Any}",
+          "null_value": "Dict{String, Any}()",
+          "default": "Dict{String, Any}()"
+        },
+        {
+          "name": "states",
+          "comment": "",
+          "internal_default": "[:δ, :ω]",
+          "data_type": "Vector{Symbol}"
+        },
+        {
+          "name": "n_states",
+          "comment": "",
+          "internal_default": 2,
+          "data_type": "Int64"
```

Comment:
I am not sure if we can implement (not urgent) that but it should be less than rating in the internal ThermalStandard